### DISCUSSION TITLE
MeshLab Server on Linux. HowTo ?

### DISCUSSION THREAD

**Discussion by djStolen** [cnr-isti-vclab/meshlab#1062] [General] [answered]
Unfortunately, found no information on this.

Could someone provide some documentation on this topic?

Thanks

**Comment by alemuntoni** [comment, marked as answer]
It is actually full of information about this just everywhere:
#911 
https://www.meshlab.net/#news
#843
#943

**Reply by djStolen** [reply]
Thanks. Just came to the world of MeshLab, so didnt search previous version. :-) Was googling though, didnt find these info though.
Maybe not bad to have a short hint/info on website referencing to this info.

Thanks for the tool. It is awesome.
